completions/zsh/_aur.zsh

```diff
@@ -44,7 +44,7 @@ __aur_list_local_packages() {
 # Helper to list valid repo-add attributes
 __aur_list_attributes() {
     declare -a attrs
-    attrs=( $(aur repo --list-attr -Sd core 2>/dev/null ) )
+    attrs=( $(aur repo --list-attr 2>/dev/null ) )
```

Comment:
While this might be safe for word splitting (you might want to set IFS explicitly to a space) it still undergoes file globbing and so either use `set -f` or use `mapfile` instead, you end up using a subshell either way:

```sh
mapfile -t attrs < <(aur repo --list-attr 2>/dev/null)
```

Reply:
Done